PowerballApi.Tests/UnitTests.csproj

```diff
@@ -78,6 +78,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```

Comment:
"None" tags mean don't build this file and don't add it to the published content. If these were "Content" tags, it would add them to the build and publish them. Please confirm these files shouldn't be built/published.

Reply:
I'm going to say that these should remain as 'None'. They're part of the unit testing suite, so that's not getting published with the project. If this were in the app, I'd consider it as content, just like the web.config is. 

Agree/Disagree?

Reply:
I agree. You're right here.